sync-core/src/main/java/com/cloudant/http/HttpConnectionManager.java

```diff
@@ -0,0 +1,47 @@
+package com.cloudant.http;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+/**
+ * Created by tomblench on 23/03/15.
+ */
+public class HttpConnectionManager {
+
```

Comment:
I can see a use for helper methods here like `HttpConnectionManager.GET(URL url, String contentType)` or `HttpConnectionManager.PUT(URL url, String contentType)`.

I'd also consider shortening the class name to just `Http` because `Http.PUT(URL url, String contentType)` looks quite clear.

However, these names might be a bit misleading in the current implementation because the methods return a connection object whereas the methods look like they carry out the request.